pkg/util/kubernetes/apiserver/leaderelection/leaderelection_test.go

```diff
@@ -326,16 +324,6 @@ func TestGetLeaderIPFollower_ConfigMap(t *testing.T) {
 	ip, err := le.GetLeaderIP()
 	assert.NoError(t, err)
 	assert.Equal(t, "1.1.1.2", ip)
-
-	// Remove bar from endpoints
```

Comment:
Why were these removed? I think that if we clean the cache before the test it should behave as before and the test should still be valid.

Reply:
I reintroduced them in the last commit